setup.cfg

```diff
@@ -72,6 +72,9 @@ include =
 console_scripts =
     procrastinate = procrastinate.cli:main
 
+[options.package_data]
+procrastinate = py.typed
```

Comment:
Why do we need that with the MANIFEST.in ?

Reply:
Yes you're right. I added it to MANIFEST.in after and forgot to remove here. Done